ansible_bender/cli.py

```diff
@@ -318,12 +319,13 @@ def _list_builds(self):
             build_time = ""
             if b.build_finished_time and b.build_start_time:
                 build_time = b.build_finished_time - b.build_start_time
+                build_time = fancy_time(build_time)
```

Comment:
I'm trying this now and getting:
```
Traceback (most recent call last):
  File "ansible_bender/cli.py", line 400, in <module>
    main()
  File "ansible_bender/cli.py", line 396, in main
    sys.exit(CLI().run())
  File "ansible_bender/cli.py", line 366, in run
    self._list_builds()
  File "ansible_bender/cli.py", line 322, in _list_builds
    build_time = fancy_time(build_time)
  File "/home/tt/g/TomasTomecek/ansible-bender/ansible_bender/utils.py", line 271, in fancy_time
    f_time = build_time.split(":")
AttributeError: 'datetime.timedelta' object has no attribute 'split'
```

while fancy_time expects to get a string. It seems like that the best design would be if fancy_time accepted timedelta and was able to work with it instead of parsing strings (which can easily be prone to errors)

Reply:
Done 😄